engines/python/setup/djl_python/huggingface.py

```diff
@@ -242,6 +242,41 @@ def inference(self, inputs: Input) -> Output:
             return self._dynamic_batch_inference(parsed_input.batch, errors,
                                                  inputs, outputs, requests)
 
+    def add_lora(self,
+                 lora_name: str,
+                 lora_path: str,
+                 long_lora_max_len: Optional[int] = None,
+                 base_model_name: Optional[str] = None):
+        if is_rolling_batch_enabled(self.hf_configs.rolling_batch):
+            return self.rolling_batch.add_lora(lora_name, lora_path,
+                                               long_lora_max_len,
+                                               base_model_name)
+        else:
```

Comment:
do we need to support all of these apis for huggingface accelerate (non-rolling batch?)

Reply:
Removed for non rolling batch.